src/utils/hooks/useColonyReputation.ts

```diff
@@ -0,0 +1,14 @@
+import { useColonyReputationQuery } from '~data/index';
+import { Address } from '~types/index';
+
+export const useColonyReputation = (colonyAddress: Address) => {
+  const { data, error } = useColonyReputationQuery({
+    variables: {
+      address: colonyAddress,
+    },
+  });
+
+  return {
+    isColonyReputationZero: data?.colonyReputation === '0' || error,
```

Comment:
I don't like the name of this prop.

I would have gone with `colonyHasReputation`, but it's your call I guess

Reply:
done